x/gamm/types/marshal_test.go

```diff
@@ -0,0 +1,96 @@
+package types
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	yaml "gopkg.in/yaml.v2"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func TestPoolAccountMarshalYAML(t *testing.T) {
+	var poolId uint64 = 10
+
+	ymlAssetTest := []PoolAsset{
+		{
+			Weight: sdk.NewInt(200),
+			Token:  sdk.NewCoin("test2", sdk.NewInt(50000)),
+		},
+		{
+			Weight: sdk.NewInt(100),
+			Token:  sdk.NewCoin("test1", sdk.NewInt(10000)),
+		},
+	}
+	pacc, err := NewPoolAccount(poolId, PoolParams{
+		SwapFee: defaultSwapFee,
+		ExitFee: defaultExitFee,
+	}, ymlAssetTest, defaultFutureGovernor, defaultCurBlockTime)
+	require.NoError(t, err)
+
+	bs, err := yaml.Marshal(pacc)
+	require.NoError(t, err)
+
+	want := `|
+  address: cosmos1m48tfmd0e6yqgfhraxl9ddt7lygpsnsrhtwpas
+  public_key: ""
+  account_number: 0
+  sequence: 0
+  id: 10
+  pool_params:
+    swap_fee: "0.025000000000000000"
+    exit_fee: "0.025000000000000000"
+    smooth_weight_change_params: null
```

Comment:
Can we add a test for a yaml output with lbp params?

Reply:
Added